packages/react/package.json

```diff
@@ -70,7 +70,6 @@
     "@react-aria/radio": "^3.10.0",
     "@react-aria/ssr": "^3.9.1",
     "@react-aria/switch": "^3.6.0",
-    "@react-aria/textfield": "^3.14.1",
```

Comment:
I noticed that @react-aria/textfield has been removed. Could you please share the reasoning behind this decision?

Reply:
To provide simpler, lighter, and more extensible components, we avoid using libraries except in complex cases.

Reply:
I understand Shu's policy. 
I agree that the 20kb size of @react-aria/textfield is quite large for its functionality.
However, determining the complexity and investigating the effective use of WAI-ARIA for creating and modifying each component will take considerable time. It is essential to closely examine the usage of aria-live, aria-errormessage, aria-describedby, and aria-invalid in this PR review.

This investigation will indeed improve developers' understanding of WAI-ARIA but may also lead to confusion due to implementation errors and increased initial implementation costs.
Given that react-aria is already widely used, I believe it would be better to continue using it. What do you think?

Reply:
In my opinion, hiding the implementation details related to accessibility can reduce bugs and maintenance costs, which I consider to be major benefits. However, we are not fully utilizing the react-aria code. For example, useTextField has validation features internally, but these are not provided by the charcoal API.
I don't intend to remove all react-aria components. I would like to continue using more complex ones like useOverlay.
Their insights on the behavior of [defaultValue](https://github.com/adobe/react-spectrum/blob/2bda4c971dd557be7ecc44614569a8281e6cb6dd/packages/%40react-aria/textfield/src/useTextField.ts#L147) and [aria-errormessage](https://github.com/adobe/react-spectrum/blob/main/packages/%40react-aria/label/src/useField.ts#L41) are excellent, but I believe it can be implemented more concisely.

Reply:
I also understand that it is a large size for the functionality used in `cahrcoal`. 

If `react-aria` is only used for wai-aria assignments, I think the policy of not using it is very good